net/asterisk-16.x/patches/100-build-reproducibly.patch

```diff
@@ -0,0 +1,28 @@
+--- a/build_tools/make_build_h
++++ b/build_tools/make_build_h
+@@ -5,6 +5,14 @@ MACHINE=`uname -m  | sed 's/\\\\/\\\\\\\
+ OS=`uname -s`
+ USER=`id | awk -F")" '{print $1}'| awk -F"(" '{print $2}' | sed 's/\\\\/\\\\\\\\/g'`
+ DATE=`date -u "+%Y-%m-%d %H:%M:%S"`
++if [ -n "${SOURCE_DATE_EPOCH}" ]; then
++	# building reproducibly, faking some data
++	HOSTNAME='buildd.debian.org'
```

Comment:
Please, change the hostname to something related to openwrt.org. It could be (preferably) openwrt.org.

Reply:
Done.